tests/client/test_async_client.py

```diff
@@ -84,7 +84,7 @@ async def test_access_content_stream_response(server):
 
     assert response.status_code == 200
     with pytest.raises(httpx.ResponseNotRead):
-        response.content
+        response.content  # noqa: B018
```

Comment:
How can I track down what a B018 is?

Reply:
It's on the rules page: https://beta.ruff.rs/docs/rules/#flake8-bugbear-b

Should I add a comment here or ignore it globally?